### DISCUSSION TITLE
Compatibility with PostgreSQL protocol ?

### DISCUSSION THREAD

**Discussion by shubhamranjan** [ClickHouse/ClickHouse#29705] [Q&A] [answered]
https://github.com/ClickHouse/ClickHouse/blob/master/programs/server/config.xml#L93

The line here mentions compatibility with PostgreSQL protocol  but the [hosted docs](https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings/) don't seem to mention anything about it. Is it something still under development ?

**Comment by alexey-milovidov** [comment, marked as answer]
It is ready for evaluation and usage. It is currently not under active development.
The compatibility may not be 100% complete, but if your application works fine, it can be used in production.

We are collecting the feedback.

It is only for protocol-level compatibility. Query level compatibility is out of scope. 
You can connect with PostgreSQL tools and drivers using PostgreSQL protocol, but you have to do queries in ClickHouse syntax.

There is no documentation because we did not write it.
